### DISCUSSION TITLE
Allure report generation if a tag fails in the BeforeAll hook in cucumber framework

### DISCUSSION THREAD

**Discussion by devrajshetake** [allure-framework/allure2#2410] [Questions & Support] [answered]
I am using Cucumber and Playwright with typescript to run my BDD tests. The allure report I am generating contains all the executed scenarios, but it does not contain the scenarios which are not executed. A `BeforeAll` hook runs before execution of all the scenarios within a tag, the problem arises when the `BeforeAll` hook fails for some reason, all the scenarios within this tag wont be executed, so they wont be mention in the report. Due to this, the total number of scenarios within my test suit are being displayed less in the report. I want the unexecuted scenarios to be mentioned in the allure report. Is there a way to do so? Please help.

**Comment by baev** [comment]
Seems like a feature request

**Reply by baev** [reply, marked as answer]
https://github.com/allure-framework/allure-js/issues/907

**Reply by baev** [reply]
https://github.com/allure-framework/allure-js/issues/907#issuecomment-2194432479
